test/Performance/HelloWorld.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
```

Comment:
The way I'm envisioning this is that this file represents "perf tests against hello world".  So, basically the latter option in your comment.